index.html

```diff
@@ -0,0 +1,224 @@
+            <ul>
+                <li style="display:inline"><a href="#">HOME</a></li>
+                <li style="display:inline"><a href="#">SERVIÇOS</a></li>
+                <li style="display:inline"><a href="#">PORTFÓLIO</a></li>
+                <li style="display:inline"><a href="#">SOBRE MIM</a></li>
+                <li style="display:inline"><a href="#">CONTATO</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section id="banner">
+        <h1>BEM VIND@</h1>
+        <h2>Eu sou Daiane Alves.</h2>
+        <p>
+            Analista de sistemas na Lyseon Tech, organizadora da comunidade PHPRio(comunidade de desenvolvedores PHP do
+            Rio de Janeiro), fundadora da comunidade PHPWomenRJ (maior comunidade de mulheres cis e trans que
+            desenvolve PHP do Brasil) e coordenadora local da comunidade WomakersCode(comunidade de emponderamento
+            feminino).
+        </p>
+        <div id="primeiro-slide">
+            <figure>
+                <img src="./images/slider01.png">
+            </figure>
+            <p>
+                OUTRO TEXTO AQUI!
+            </p>
+        </div>
+
+        <div id="segundo-slide">
+            <figure>
+                <img src="./images/Slider02.png">
+            </figure>
+            <p>
+                OUTRO TEXTO AQUI!
+            </p>
+        </div>
+
+        <div id="terceiro-slide">
+            <figure>
+                <img src="./images/Slider03.png">
+            </figure>
+            <p>
+                OUTRO TEXTO AQUI!
+            </p>
+        </div>
+    </section>
+
+    <section id="servicos">
+        <h1>Serviços</h1>
+        <p>Precisando tirar aquela ideia do papel? Procurando um bom profissional para desenvolver seu site? Fale
+            comigo!</p>
+        <div id="linha">
+            <hr />
+        </div>
+
+        <div class="row">
+            <div class="coluna-direita">
+                <img src="./images/Service1.png" alt="servico1">
+                <p>Descrição Serviços 1</p>
+            </div>
+
+            <div class="coluna-centro">
+                <img src="./images/Service2.png" alt="servico2">
+                <p>Descrição Serviços 2</p>
+            </div>
+
+            <div class="coluna-esquerda">
+                <img src="./images/Service3.png" alt="servico3">
+                <p>Descrição Serviços 3</p>
+            </div>
+        </div>
+    </section>
+
+    <section id="portfolio">
+        <h1>Meus Trabalhos</h1>
+        <p>Confira alguns dos meus trabalhos realizados!</p>
+
+        <div id="linha">
+            <hr />
+        </div>
+
+        <nav id="trabalhos">
+            <ul>
+                <li style="display:inline"><a href="#">Tudo</a></li>
+                <li style="display:inline"><a href="#">Website</a></li>
+                <li style="display:inline"><a href="#">Template</a></li>
+                <li style="display:inline"><a href="#">Tema</a></li>
+            </ul>
+
```

Comment:
Faltou fechar o elemento `<nav>`

```suggestion
</nav>
```